File: Codechef/Practice/Oneful Pairs.cpp
https://www.codechef.com/problems/ONEFULPAIRS




#include <iostream>
using namespace std;

int main() {
	// your code goes here
	int a,b;
	cin >> a >> b;
	(a + b +(a*b) == 111) ? cout << "YES" : cout << "NO";
	cout << endl;
	return 0;
}

File: Codechef/Practice/Oneful Pairs.go
https://www.codechef.com/practice/course/go/PGOL03/problems/ONEFULPAIRS?tab=statement




package main
import "fmt"

func main(){
    var a,b int
    fmt.Scan(&a)
    fmt.Scan(&b)
    if(a+b+(a*b)==111){
        fmt.Println("YES")
    } else{
        fmt.Println("NO")
    }
}